azDevOps/azure/templates/v2/scripts/inject-secrets-from-env-var.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+secretfolderpath=$1
+
+cd $secretfolderpath
+echo "Scanning for files in $secretfolderpath"
+echo "Found these files:"
+ls -lp1
+
+for file in $(find -type f); do
+
+    envVar=$(echo "${file#"./"}" | tr / _);
```

Comment:
We need to add extra logic to this script to handle case sensitive values
Currently, need to be an exact match between the filename and the ENV VAR name, that is generally uppercase
We should look for an ENV var as UPPERCASE instead of the name of the file